client/state/partner-portal/licenses/hooks/use-bundle-licenses-query.ts

```diff
@@ -1,21 +1,23 @@
-import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
 import { useTranslate } from 'i18n-calypso';
-import { useEffect } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { wpcomJetpackLicensing as wpcomJpl } from 'calypso/lib/wp';
 import { useDispatch } from 'calypso/state';
 import { License } from 'calypso/state/partner-portal/types';
 import { errorNotice } from '../../../notices/actions';
 import { formatLicenses } from '../handlers';
 
-export default function useBundleLicensesQuery(
-	parentLicenseId: number,
-	perPage: number = 100
-): UseQueryResult< License[] > {
+export default function useBundleLicensesQuery( parentLicenseId: number, perPage: number = 25 ) {
+	const [ licenses, setLicenses ] = useState< License[] >( [] );
+	const [ total, setTotal ] = useState< number >( 0 );
+	const [ page, setPage ] = useState< number >( 1 );
+	const [ fetching, setFetching ] = useState< boolean >( false );
```

Comment:
Do we need to set a state for this? Can't we use `isLoading` instead?

Reply:
You are right. `isLoading` should work 👍 thanks for the suggestion.